client/blocks/inline-help/popover-content.scss

```diff
@@ -0,0 +1,168 @@
+.inline-help__popover-content {
+	background-color: var( --color-surface );
+
+	p {
+		font-size: $font-body-extra-small;
+
+		@include breakpoint-deprecated( '>660px' ) {
+			font-size: $font-body-small;
+		}
+	}
+
+	a,
+	button {
+		font-size: $font-body-extra-small;
+
+		@include breakpoint-deprecated( '>660px' ) {
+			font-size: $font-body-small;
+		}
+	}
+
+	&.is-secondary-view-active {
+		.inline-help__search {
+			display: none;
+		}
+
+		.inline-help__secondary-view {
+			display: block;
+		}
+
+		.inline-help__footer {
+			.inline-help__more-button,
+			.inline-help__contact-button {
+				display: none;
+			}
+
+			.inline-help__cancel-button {
+				display: block;
+			}
+		}
+	}
+}
+
+.inline-help__secondary-view {
+	display: none;
+
+	box-sizing: border-box;
+	max-height: 70vh;
+	min-height: 206px;
+	overflow-y: auto;
+	padding: 16px;
+
+	.button {
+		width: 100%;
+		text-align: center;
+	}
+}
+
+.inline-help__view-heading {
+	display: block;
+	font-size: $font-body-small;
+	font-weight: 600;
+	margin-bottom: 5px;
+}
+
+.inline-help__richresult {
+	text-align: left;
+
+	.inline-help__richresult__title {
+		display: block;
+		font-size: $font-body-small;
+		font-weight: 600;
+		margin-bottom: 5px;
+	}
+}
+
+.inline-help__forum-view {
+	text-align: left;
+}
+
+.inline-help__contact {
```

Comment:
At first glance, it appears that this classname is not used anywhere, but at a second glance it appears to be used here:

https://github.com/Automattic/wp-calypso/blob/a82f8469a286519ffe3e68793b5d61778cf34e07/client/blocks/inline-help/popover-content.jsx#L147